### ISSUE TITLE
Website: have the search bar follow you as you scroll

### ISSUE THREAD

**Issue by dieps** [fabd/kanji-koohii#64]
Would be convenient to be able to advance/search flashcards after looking at stories on the website
![delete later 333](https://cloud.githubusercontent.com/assets/13320702/22585308/3eb3e9b6-e99b-11e6-9110-9b8f98db8a5a.png)

**Comment by fabd** [comment]
Yes I agree. I'm not marking it NEXT yet because it's less critical but I've been thinking about it.

I think for mobile it would be good too. On mobile it's easier to figure out (bar implementation details), since the search is a full width bar under the top nav, I could make it stick there.

On desktop it's less straightforward. Making the box in the sidebar fixed as you suggest is one approach. Perhaps making the whole sidebar on desktop fixed? THis migth be easier. Otherwise the search box will overlap other elemetns when you start scrolling down.
